zio-http-example/src/main/scala/example/AuthenticationServer.scala

```diff
@@ -39,19 +40,19 @@ object AuthenticationServer extends ZIOAppDefault {
   }
 
   // Http app that is accessible only via a jwt token
-  def user: UHttpApp = Http.collect[Request] { case Method.GET -> !! / "user" / name / "greet" =>
+  def user: HttpRoute[Any, Nothing] = Http.collect[Request] { case Method.GET -> !! / "user" / name / "greet" =>
```

Comment:
Maybe we can keep the old type aliases to avoid the updating code??

Reply:
I think these old aliases are conflicting with the new ones:

```scala
type HttpRoute[-R, +Err] = Http[R, Err, Request, Response]
type App[-R]             = HttpRoute[R, Response]
```

Probably the `HttpRoute` name is not very good either anymore (I defined it when it was `Route[R, Err, Req, Resp]`).

So we should review all the names together. I don't like much these old aliases but maybe it is more important to keep them for people who already use them.